src/KristofferStrube.Blazor.Popper/KristofferStrube.Blazor.Popper.csproj

```diff
@@ -25,12 +25,18 @@
       <PackagePath>\</PackagePath>
     </None>
   </ItemGroup>
-
   <ItemGroup>
     <SupportedPlatform Include="browser" />
   </ItemGroup>
-  
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.1" />
-  </ItemGroup>
-</Project>
+	<ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">
+		<PackageReference Include="Microsoft.AspNetCore.Components.Web" Version="6.0.24" />
```

Comment:
Changing the patch version of `Microsoft.AspNetCore.Components.Web` for people using .NET 6 could mean that they would need to upgrade their version. This has no benefit to the project so it simply limits who can consume it. People consuming the library can always use a greater version if they prefer.